### DISCUSSION TITLE
关于配置同步/备份的构想

### DISCUSSION THREAD

**Discussion by kytrun** [ReadYouApp/ReadYou#6] [Ideas]
安卓没有苹果生态的 iCloud API 同步，设想通过 WebDAV 备份各种 RSS 云服务 API 的配置、设置项以及已读文章。

大致方案：本地 RSS 在多设备下各自拉取，仅自动标记在其他设备`已读`状态文章的 url。因不断累积这个列表会越来越大，可以考虑开放设置一定时间后的自动标记，如 `n` 天以前的文章直接设为已读，清除列表中超时的记录。

如果制定了较好的方案和格式，在如 Windows APP / Browser Extension 上可以通用，Windows 11 也可以用安卓子系统。

请开发者考虑，欢迎探讨~

**Comment by Ashinch** [comment]
是的，像 FreshRSS 之类的自托管服务做的就是这样的工作，让多设备可以同步相同的阅读进度。

Reeder 使用了 iCloud 来达成这个目的，所以在 Android 中确实也可以通过 WebDAV 来完成。

ReadYou 会优先支持像 FreshRSS 这样的、现成的、完备的自托管服务，而通过 WebDAV 来达成同步阅读进度的优先级是在这之后。
